apps/remix-ide/src/app/tabs/compileTab/compileTab.js

```diff
@@ -109,7 +109,11 @@ class CompileTab extends Plugin {
         `
         const configFilePath = 'remix-compiler.config.js'
         this.fileManager.setFileContent(configFilePath, fileContent)
-        this.call('hardhat', 'compile', configFilePath)
+        this.call('hardhat', 'compile', configFilePath).then((result) => {
```

Comment:
I thought we could so something like `this.call('terminal', 'log', result)` instead of using event, but that works too, will add @bunsenstraat for review.

Reply:
Terminal plugin is not exposing any methods: https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/src/app/panels/terminal.js#L29

Reply:
we have a pr from an external dev which expose terminal API, that maybe time for the terminal to have a minimal API.
let see what @bunsenstraat think

Reply:
https://github.com/ethereum/remix-project/pull/1210

this.call('terminal', 'log', result) will now work, it will also deal with \n and so on